crates/store/src/db/sql.rs

```diff
@@ -396,15 +401,14 @@ pub fn insert_notes(transaction: &Transaction, notes: &[NoteRecord]) -> Result<u
     let mut count = 0;
     for note in notes.iter() {
         let details = note.details.as_ref().map(|details| details.to_bytes());
-
         count += stmt.execute(params![
             note.block_num,
             note.note_index.batch_idx(),
             note.note_index.note_idx_in_batch(),
             note.note_id.to_bytes(),
-            note.note_type as u8,
-            u64_to_value(note.sender),
-            note.tag,
+            note.metadata.note_type() as u8,
+            u64_to_value(note.metadata.sender().into()),
+            note.metadata.tag().inner(),
```

Comment:
Seems like we are not storing `metadata.aux` field in the database yet. We should probably make an issue to do that (unless one already exists?).

Reply:
Created https://github.com/0xPolygonMiden/miden-node/issues/342